packages/shopping/src/application.ts

```diff
@@ -99,4 +101,15 @@ export class ShoppingApplication extends BootMixin(
 
     this.bind(UserServiceBindings.USER_SERVICE).toClass(MyUserService);
   }
+
+  modifyApiSpecAndRebind() {
+    let openAPISchema = this.getSync(RestBindings.API_SPEC);
+    addSecuritySchemesToOpenApiSpec(openAPISchema);
+    this.bind(RestBindings.API_SPEC).to(openAPISchema);
+  }
+
+  async start(): Promise<void> {
+    this.modifyApiSpecAndRebind();
```

Comment:
We should add a comment claiming this is a workaround before an extension point for open api schema generation is introduced.

Reply:
This code will be obsolete in my next commit; based on our conversation yesterday. 
But I definitely will provide a comment above the `this.api()` call. :)